src/main/kotlin/org/jitsi/jicofo/JicofoServices.kt

```diff
@@ -247,6 +248,7 @@ open class JicofoServices {
         jigasiDetector?.let { put("jigasi_detector", it.stats) }
         putAll(ColibriConferenceImpl.stats.toJson())
         put("threads", ManagementFactory.getThreadMXBean().threadCount)
+        put("jingle", AbstractOperationSetJingle.getStats())
```

Comment:
Does this need a `toJson()` on it?

Reply:
No, it's exposed as a `JSONObject` already.